index.html

```diff
@@ -42,46 +34,180 @@ <h2 class="title-about">I'm a Software Developer</h2>
             don’t hesitate to contact me.
         
         </p>
-
-        <div class="deco1"></div>
         
 
         <ul class="social-icon">
             <li>
                 <a href="https://github.com/ulises2607" class="soc-icon">
-                    <img src="Images/Social/GitHub.svg" alt="github-profile">
+                    <img src="Images/Main/Social/GitHub.svg" alt="github-profile">
                 </a>
             </li>
             <li>
                 <a href="https://www.linkedin.com/in/cesar-ulises-cappa-subelza/" class="soc-icon">
-                    <img src="Images/Social/Linkedin.svg" alt="linkedin-profile">
+                    <img src="Images/Main/Social/Linkedin.svg" alt="linkedin-profile">
                 </a>
             </li>
             <li>
                 <a href="https://www.angellist.com/" class="soc-icon">
-                    <img src="Images/Social/Angelist.svg" alt="Angelist-mainpage">
+                    <img src="Images/Main/Social/Angelist.svg" alt="Angelist-mainpage">
                 </a>
             </li>
             <li>
                 <a href="www.twitter.com" class="soc-icon">
-                    <img src="Images/Social/Twitter.svg" alt="twitter-mainpage">
+                    <img src="Images/Main/Social/Twitter.svg" alt="twitter-mainpage">
                 </a>
             </li>
             <li>
                 <a href="https://medium.com/" class="soc-icon">
-                    <img src="Images/Social/Medium.svg" alt="medium-mainpage">
+                    <img src="Images/Main/Social/Medium.svg" alt="medium-mainpage">
                 </a>
             </li>
 
         </ul>
-            
 
-        
+    </section>
+
+    <section class="recent-works">
+
+        <h1>My Recent Works</h1>
+
+
+        <img class="img-work" src="Images/Work/Work-img-1.png" alt="Multi-post-stories">
+
+        <h2>Multi-Post Stories</h2>
+
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, when an unknown printer took 
+            a standard dummy text.
+        </p>
+
+        <ul class="tech">
+            <li><a href="#">CSS</a></li>
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+        </ul>
+
+        <a href="#" class="btn-works-1">See Projects</a>
+    </section>
+
+<!-- Project Cards -->
+    <section>
+        <div class="card-container">
+            <div class="card">
+                <h1 class="card-title">Profesional Art Printing Data</h1>
```

Comment:
- [ ] **[OPTIONAL]** Awesome job incorporating the suggested changes from the previous reviewer! I just wanted to offer a friendly suggestion: it would be great if you could try to avoid using multiple `h1` tags in your project. By doing so, you **can enhance the overall readability and structure of your content.** Not only will this make **it easier for users to navigate through your project, but it will also improve the accessibility for individuals with visual impairments who rely on assistive technologies.** 😉